src/components/navigation/left/ServerListSidebar.tsx

```diff
@@ -320,9 +320,13 @@ export const ServerListSidebar = observer(({ unreads, lastOpened }: Props) => {
             {!isTouchscreenDevice && (
                 <SettingsButton>
                     <Link to="/settings">
-                        <IconButton>
-                            <Cog size={32} strokeWidth="0.5" />
-                        </IconButton>
+                        <Tooltip
+                            content="Settings"
```

Comment:
Needs i18n.
`<Text id="app.settings.title" />` instead of "Settings" as content.
`Text` can be imported from `preact-i18n`